test/config-luatest/cluster_config_schema_test.lua

```diff
@@ -100,6 +100,13 @@ g.test_cluster_config = function()
     t.assert_equals(result, expected)
 end
 
+local function check_enterprise(data_ee, data_ce)
+    if require('tarantool').package == 'Tarantool Enterprise' then
+        return data_ee
+    end
+    return data_ce
+end
```

Comment:
It feels somewhat less verbose than `<if> and true or false` because of the naming, maybe just add a variable `is_tarantool_ee` or something similar? Or just think of the better name, but I can't think of one.

Reply:
Replaced by ternary operator. Also, introduced `is_enterprise` variable, so I suggest to use it in #8883.

Reply:
Thanks, will do.